src/Tribe/Admin/Commerce/Warnings.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Conditional Warnings on Ticket Features.
+ */
+
+namespace Tribe\Tickets\Admin\Commerce;
+
+/**
+ * Warnings handling class.
+ *
+ * @since TBD
+ */
+class Warnings {
+
+	/**
+	 * Hooks actions for showing warnings
+	 */
+	public function hook() {
+		add_action( 'tribe_events_tickets_new_ticket_buttons', [ $this, 'recurring_event_warning_msg' ] );
+	}
+
+	/**
+	 * Create Recurring Event Warning Message Label
+	 *
+	 * @since TBD
+	 *
+	 * @param int $post_id Post ID.
+	 */
+	public function recurring_event_warning_msg( $post_id ) {
+		if ( ! class_exists( 'Tribe__Events__Pro__Main' ) ) {
+			return;
+		}
+
+		if ( ! function_exists( 'tribe_is_recurring_event' ) ) {
+			return;
+		}
+
+		if ( ! tribe_is_recurring_event( $post_id ) ) {
+			return;
+		}
+
+		$warning = __( 'This is a recurring event. If you add tickets, they will only show up on first event in the recurrence series. Please carefully configure your recurring events.', 'event-tickets' );
+
+		$this->show_notice( $warning );
+	}
+
+	/**
+	 * Render the notice block.
+	 *
+	 * @since TBD
+	 *
+	 * @param string $msg Message to show.
+	 * @param string $type Type of message.
+	 */
+	public function show_notice( $msg, $type = 'info' ) {
+		$icon = 'dashicons-' . $type;
+		?>
+		<div class="ticket-editor-notice <?php echo esc_attr( $type ); ?>">
+			<span class="dashicons <?php echo esc_attr( $icon ); ?>"></span>
+			<span class="message"><?php esc_html_e( $msg, 'event-tickets' ); ?></span>
+		</div>
```

Comment:
:no_entry_sign: The $text arg must be a single string literal, not "$msg".

<sub>hash: 2d106cf6a6d93795625f51962d7c7614</sub>